File: .cpp/Program/Storage.cpp
#pragma once
#include <bits/stdc++.h>
#include "Component.cpp"
using namespace std;

class Storage : public Component {
    private:
        int size;

    public:
        Storage () { }

        Storage (string brand, string model, int size) : Component (brand, model) {
            this->size = size;
        }

        void setSize (int size) { this->size = size; }

        int getSize () { return size; }

        virtual void info () override { cout << getBrand() << " " << getModel() << " " << size << "GB"; }

        virtual ~Storage () { }
};

File: .py/Program/Storage.py
from Component import Component

class Storage (Component):
    __size = 0

    def __init__ (self, brand = "", model = "", size = 0):
        super ().__init__ (brand, model)
        self.__size = size

    def setSize (self, size):
        self.__size = size

    def getSize (self):
        return self.__size
    
    def info (self):
        print (f"{ self.getBrand() } { self.getModel() } { self.getSize() }GB", end = "")